docs/include/content.tex

```diff
@@ -4,26 +4,52 @@ \chapter{Introduction}
 
   
 \chapter{Requirements} 
-    The microservice will be about a calculation process for ball bearings.  
+    The microservice will be about a calculation process for ball bearings in O-assignment. 
 
     \section{Flow chart for calculation process}
-        \begin{tikzpicture}[node distance = 3cm]
+        Bearing A and B (Kegelrollenlager) in O-assignment. 
+        
+        \begin{tikzpicture}[node distance = 2cm]
             \node [terminator, fill=blue!20] (start) {\textbf{Start}};
-            \node [data, fill=blue!20, below of=start] (data) {Provide data};
-            \node [decision, fill=blue!20, below of=data] (decision) {Valid data?};
-            \node [process, fill=red!20, right of=decision] (error) {Error};
-            \node [process, fill=green!20, below of=decision] (success) {Success};
-            \node [terminator, fill=blue!20, below of=success] (end) {\textbf{End}};
-            \node[draw=none] at (1.70, -5.75) (no) {No};
-            \node[draw=none] at (0.35, -7.80) (yes) {Yes};
-            \path [connector] (start) -- (data);
-            \path [connector] (data) -- (decision);
-            \path [connector] (decision) -- (error);
-            \path [connector] (decision) -- (success);
-            \path [connector] (error) |- (end);
-            \path [connector] (success) -- (end);
+            \node [decision, fill=blue!20, below of=start, yshift=-2cm] (decision) {is Einschaltdauer set?};
```

Comment:
May you remove this decision ? We will ignore the influence of Einschaltdauer and will focus on the calculation of the actual bearing life. Ignore means setting it in mind to 100 % so that you can remove it from the chart and from the following implementation.

Reply:
Okay, that's fine for me.